tests/test_var_csv.py

```diff
@@ -0,0 +1,36 @@
+""" This script runs tests using the variable descriptions csv file and the catalog contents variable choices. 
+This script tests that the variables and descriptions listed in the catalog are the same as (or a smaller subset of) the variables and descriptions listed in the csv file. 
+It also tests that the variable descriptions csv file contains the expected column names. 
+"""
+
+import pandas as pd
+import pytest
+from climakitae.selectors import CatalogContents
+import pkg_resources
+DATA_PATH = pkg_resources.resource_filename('climakitae', 'data/')
+CSV_FILE = pkg_resources.resource_filename('climakitae', 'data/variable_descriptions.csv')
+
+@pytest.fixture
+def cat_contents():  
+    """Load variable choices listed in Analytics Engine catalog. """
+    return CatalogContents()._variable_choices
+
+@pytest.fixture
+def descrip_dict_formatted():
+    """Read in csv file formatted with the same columns as CatalogContents()._variable_choices. """
+    csv = pd.read_csv(CSV_FILE, index_col="description", usecols=["name","description"])
+    return csv.to_dict(orient="dict")["name"]
+
+@pytest.fixture
+def descrip_pd(): 
+    """Read in csv file as pandas dataframe object. """
+    pd_df = pd.read_csv(CSV_FILE)
+    return pd_df 
+
+def test_csv_matches_cat_contents(cat_contents, descrip_dict_formatted):
+    """Ensure that items in catalog are the same as (or a smaller subset of) the items in the variable csv. """
+    assert cat_contents["hourly"]["Dynamical"].items() <= descrip_dict_formatted.items(), "Catalog contents are not the same as (or a smaller subset of) the items in the variable descriptions csv."
```

Comment:
Hmm I just tried running this locally and this test failed for me. The first thing I noticed is one item in `cat_contents["hourly"]["Dynamical"]` is `'Precipitation (total)': ''`
whereas in the `descrip_dict_formatted` it is `'Precipitation (total)':'TOT_PRECIP'`
Happy to look into this more if it would be helpful, but you know this code better so will probably know what this is right away?

Reply:
Yeah this test doesn't pass at the moment. This is due to a bug in main (the variable "Precipitation (total)" is lacking a variable name). This bug will be fixed once the add_var_test branch is merged, and the test will pass! Kind of cool that the unit test catches the bug, though :D

Reply:
great!

Reply:
suggested order of variables:

2m temp
ppt
rh
wind_mag
wind_dir
u10
v10
PSFC
SWDIFF

maybe put the precip components last?

after that doesn't matter

Reply:
nice job on the radiation description wording - looks like these are proper atmospheric science definitions.  suggest a cross-check with WRF handbook to ensure that the model world is doing the same thing as is expected (especially the comment about clouds catching reflected radiation - want to make sure this is true in WRF)

Reply:
The tests are all passing now!!!!! WOOHOOOO!!!